client/src/components/FeedbackObject/index.js

```diff
@@ -7,159 +7,148 @@ import Button from "../Button/Button";
-					if (property === "module") {
-						return (
-							<div className="previous-feedback-module" key={index}>
-								<p className="feedback-input-head">Module:</p>
-								{!isEditing
-									? <p>{currentFeedback[property]}</p>
-									: <select
-										name={property}
-										value={currentFeedback[property]}
-										onChange={handleEdit}>
-										{modules.map((module, index) =>
-											<option key={index} value={module.name}>{module.name}</option>
-										)}
-									</select>}
-							</div>
-						);
-					}
-					if (property === "title") {
-						return (
-							<div className="previous-feedback-title" key={index}>
-								<p className="feedback-input-head">Title:</p>
-								<textarea
-									className="feedback-title-input"
-									name={property}
-									value={currentFeedback[property]}
-									onChange={handleEdit}
-									disabled={!isEditing}
-								/>
-							</div>
-						);
-					}
-					if (property === "text") {
-						return (
-							<div className="previous-feedback-text" key={index}>
-								<p className="feedback-input-head">Feedback:</p>
-								<textarea
-									className="previous-feedback-text-input"
-									name={property}
-									value={currentFeedback[property]}
-									onChange={handleEdit}
-									disabled={!isEditing}
-								/>
-							</div>
-						);
-					}
-					if (property === "mentor") {
-						return (
-							<div className="previous-feedback-mentor" key={index}>
-								<p className="feedback-input-head">Given by:</p>
-								<input
-									className="previous-feedback-mentor-input"
-									name={property}
-									value={currentFeedback[property]}
-									onChange={handleEdit}
-									disabled
-								/>
-							</div>
-						);
-					}
-					if (property === "time") {
-						return (
-							<div className="previous-feedback-time" key={index}>
-								<input
-									className="previous-feedback-time-input"
-									name={property}
-									value={timeDifference(Date.now(), currentFeedback[property])}
-									onChange={handleEdit}
-									disabled
-								/>
-							</div>
-						);
-					}
-				})}
-			<div className="edit-delete-buttons">
-				<Button
-					content={isEditing ? "Save" : "Edit"}
-					handleClick={() => {
-						setIsEditing(!isEditing);
-						isEditing && saveFeedback(currentFeedback.id);
-					}}
-				/>
-				<div>
-					<Button
-						content="Delete"
-						handleClick={() => {
-							alert("DELETE");
-							handleDelete(currentFeedback.id);
-							updateFeedback();
-						}}
-					/>
-				</div>
-			</div>
-		</div>
-	);
+  return (
+    <div className="prev-feedback-list">
+      {Object.keys(currentFeedback).length &&
+        Object.keys(currentFeedback).map((property, index) => {
+          if (property === "module") {
+            return (
+              <div className="feedback-title-and-module">
+                <div className="feedback-title" key={index}>
+                    <textarea
```

Comment:
This one is better to be input rather than textarea, as the one in feedback form. I made it restricted to 50 chars and we won't need to expand it.

Reply:
Your input styles and this style here is different.

Reply:
You can just change the <textArea> to <input> and give it whatever class you want.